recipes/pipestat/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pipestat" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pipestat-{{ version }}.tar.gz
+  sha256: 986c6edd9097b22fec445c0dcb5283a4d992b9942e6df49857bf57beef5c9792
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - pipestat = pipestat.__main__:main
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - setuptools
```

Comment:
`setuptools` isn't a run dependency

Reply:
Unfortunately, the build fails and runs into a `ModuleNotFoundError: No module named 'pkg_resources'` error when I remove setuptools from the list of run dependencies.

Reply:
I see. You are right. `setuptools` isn't used directly by `pipestat`.
`eido` is using [it](https://github.com/pepkit/eido/blob/2525940bb3b424f8fae7f3b5c8f68ebc5da00f60/eido/conversion.py#L6C6-L6C19). It should be defined in that package instead of here. It's fine for now. I opened an issue: https://github.com/conda-forge/eido-feedstock/issues/16